### PR TITLE
fixup! sideComponent: Add SideComponent back

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
https://phabricator.endlessm.com/T26597

### CODE REVIEW

File: js/ui/layout.js [hunk 1]
```diff
@@ -63,7 +63,8 @@ const defaultParams = {
 };
 
 var LayoutManager = GObject.registerClass({
-    Signals: { 'hot-corners-changed': {},
+    Signals: { 'background-clicked': {},
```
Comment (cosimoc) [inline js/ui/layout.js]: This is OK, but I'm not sure that's all we need.
As mentioned on the phabricator ticket, in the eos3.5 codebase `sideComponent.js` connects to this signal.
This patch will fix the warning when the signal is emitted, but functionality-wise, on master, even after this patch I don't see anything connecting to this signal.

Comment (GeorgesStavracas) [reply js/ui/layout.js]: You're right; brought back the code that connected to it.

File: js/ui/sideComponent.js [hunk 2]
```diff
@@ -78,6 +78,15 @@ class SideComponent extends GObject.Object {
         this._propertiesChangedId =
             this.proxy.connect('g-properties-changed', this._onPropertiesChanged.bind(this));
 
+        // Clicking the background (which calls overview.showApps) hides the component,
+        // so trying to open it again will call WindowManager._mapWindow(),
+        // which will hide the overview and animate the window.
+        // Note that this is not the case when opening the window picker.
+        this._desktopShownId = Main.layoutManager.connect('background-clicked', () => {
```
Comment (cosimoc) [inline js/ui/sideComponent.js]: Is there a reason this https://github.com/endlessm/gnome-shell/blob/eos3.5/js/ui/sideComponent.js#L60 and this https://github.com/endlessm/gnome-shell/blob/eos3.5/js/ui/sideComponent.js#L110 snippets were not brought back?

Comment (GeorgesStavracas) [reply js/ui/sideComponent.js]: I did not bring them back because they are already there.

Comment (cosimoc) [reply js/ui/sideComponent.js]: Lol sorry!

**Pull request merged**